storage/plugins/archives/ArchivesPlugin.php

```
@@ -5224,6 +5271,241 @@ public function iiifCollectionAction(
+            // Table may be absent on installs where viaf-authority has never
+            // been activated; silently degrade to an empty list rather than
+            // logging a noisy error on every RiC request.
+            return $uris;
+        }
+        $stmt->bind_param('i', $authorityId);
+        $stmt->execute();
+        $result = $stmt->get_result();
+        if ($result instanceof \mysqli_result) {
+            while ($row = $result->fetch_assoc()) {
+                foreach (['viaf_uri', 'isni_uri'] as $col) {
+                    $v = $row[$col] ?? null;
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $v. Configured minimum length is 3. (undefined)

(ShortVariable)